tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php

```diff
@@ -762,13 +770,17 @@ private function provideClassyExtendingCases($classy)
                 sprintf('<?php
 %s Test124 extends
 \Exception
-{}', $classy),
```

Comment:
This `sprintf` call and the one below were not multiline and did not need to be fixed. You should either revert these changes or add the missing newline after the opening brace.